src/test/java/com/epam/ta/reportportal/entity/project/ProjectUtilsTest.java

```diff
@@ -0,0 +1,178 @@
+import com.epam.ta.reportportal.entity.enums.ProjectAttributeEnum;
+import com.epam.ta.reportportal.entity.enums.ProjectType;
+import com.epam.ta.reportportal.entity.enums.TestItemIssueGroup;
+import com.epam.ta.reportportal.entity.item.issue.IssueGroup;
+import com.epam.ta.reportportal.entity.item.issue.IssueType;
+import com.epam.ta.reportportal.entity.user.ProjectUser;
+import com.epam.ta.reportportal.entity.user.User;
+import com.google.common.collect.Sets;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.*;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+public class ProjectUtilsTest {
+
+	private Project project;
+
+	private Set<Attribute> defaultAttributes;
+
+	@Before
+	public void setUp() throws Exception {
+		project = getTestProject();
+		defaultAttributes = getDefaultAttributes();
+	}
+
+	@Test
+	public void defaultProjectAttributes() {
+		final Set<ProjectAttribute> projectAttributes = ProjectUtils.defaultProjectAttributes(project, defaultAttributes);
+		projectAttributes.forEach(it -> {
+			assertEquals(project, it.getProject());
+			assertThat(it.getAttribute()).isIn(defaultAttributes);
+			assertEquals(it.getValue(), ProjectAttributeEnum.findByAttributeName(it.getAttribute().getName()).get().getDefaultValue());
+		});
+	}
+
+	@Test
+	public void defaultIssueTypes() {
+		final List<IssueType> defaultIssueTypes = getDefaultIssueTypes();
+		final Set<ProjectIssueType> projectIssueTypes = ProjectUtils.defaultIssueTypes(project, defaultIssueTypes);
+		assertNotNull(projectIssueTypes);
+		assertEquals(defaultIssueTypes.size(), projectIssueTypes.size());
+		projectIssueTypes.forEach(it -> {
+			assertEquals(project, it.getProject());
+			assertThat(it.getIssueType()).isIn(defaultIssueTypes);
+		});
+	}
+
+	@Test
+	public void doesHaveUser() {
+		assertTrue(ProjectUtils.doesHaveUser(project, "test_user"));
+		assertFalse(ProjectUtils.doesHaveUser(project, "darth_vader"));
+	}
+
+	@Test
+	public void findUserConfigByLogin() {
+		final ProjectUser projectUser = ProjectUtils.findUserConfigByLogin(project, "test_user");
+		assertNotNull(projectUser);
+		assertEquals(getTestUser(), projectUser.getUser());
+		assertEquals(project, projectUser.getProject());
+		assertEquals(ProjectRole.PROJECT_MANAGER, projectUser.getProjectRole());
+	}
+
+	@Test
+	public void getConfigParameters() {
+		final Map<String, String> configParameters = ProjectUtils.getConfigParameters(getProjectAttributes());
+		assertNotNull(configParameters);
+		assertTrue(!configParameters.isEmpty());
+		assertThat(configParameters).isEqualTo(getAttributeWithValuesMap().entrySet()
+				.stream()
+				.collect(Collectors.toMap(it -> it.getKey().getName(), Map.Entry::getValue)));
+	}
+
+	@Test
+	public void getOwner() {
```

Comment:
?

Reply:
private static final String OWNER = "OWNER";

Reply:
it is empty here